DOCUMENTATION.md

```diff
@@ -1,810 +1,780 @@
+|`accepted`|Notify is printing and posting the letter|||Yes|
+|`received`|The provider has received the letter to deliver|||Yes|
 
-##### `template_type`
+#### templateType (optional)
 
-If omitted all messages are returned. Otherwise you can filter by:
+You can filter by:
 
 * `email`
 * `sms`
 * `letter`
 
-You can omit this argument to ignore the filter.
-
-##### `status`
+#### reference (optional)
 
-You can filter by:
+A unique identifier. This reference identifies a single unique notification or a batch of notifications.
 
-* `sending` - the message is queued to be sent by the provider.
-* `delivered` - the message was successfully delivered.
-* `failed` - this will return all failure statuses `permanent-failure`, `temporary-failure` and `technical-failure`.
-* `permanent-failure` - the provider was unable to deliver message, email or phone number does not exist; remove this recipient from your list.
-* `temporary-failure` - the provider was unable to deliver message, email box was full or the phone was turned off; you can try to send the message again.
-* `technical-failure` - Notify had a technical failure; you can try to send the message again.
+```ruby
+reference: 'your_reference_string'
+```
 
-You can omit this argument to ignore the filter.
+#### older_than (optional)
 
-##### `reference`
+Input the ID of a notification into this argument. If you use this argument, the client returns the next 250 received notifications older than the given ID.
 
-This is the `reference` you gave at the time of sending the notification. The `reference` can be a unique identifier for the notification or an identifier for a batch of notifications.
+```ruby
+older_than: 'e194efd1-c34d-49c9-9915-e4267e01e92e'
+```
 
-You can omit this argument to ignore the filter.
+If you leave out this argument, the client returns the most recent 250 notifications.
 
+The client will only return notifications that are 7 days old or less. If the notification specified in this argument is older than 7 days, the client will return an empty `collection` response.
 
-##### `older_than`
-You can get the notifications older than a given `Notification.id`.
-You can omit this argument to ignore this filter.
+### Response
 
-</details>
+If the request to the client is successful, the client will return a `Notifications::Client::NotificationsCollection` object. This object is named `response` in the method example.
 
+You must then call either the `.links` method or the `.collection` method on this object:
 
+|Method|Information|
+|:---|:---|
+|`response.links`|Returns a hash linking to the requested notifications (limited to 250)|
+|`response.collection`|Returns an array of the required notifications|
 
-## Get a template by ID
+If you call the `collection` method on this object to return an array, you will then have to call different methods on this array to return the requested information.
```

Comment:
@klssmith what are the different methods on this array that you can call to return requested information? Are they the same as other methods?

Reply:
Methods on the notification are the same as get status of one message. 
Also update Get all templates, link to get template by id / and version